test/e2e_node/node_pid_limit_test.go

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
Can we also add to the node container manager test, which inspects cgroup values to ensure we are setting them correctly?

https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/node_container_manager_test.go#L192

Reply:
Sure, I'll look at that on Monday.

Reply:
OK, done.

I note that at least on my system, at https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/node_container_manager_test.go#L195 I had to change `kubepods` to `kubepods.slice` for all references to the root cgroup (obviously, I did not push that change).  I presume that has to do with the fact that I'm using the `systemd` cgroup driver?

Reply:
yeah, it sounds like we need to do something like this for the test: https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/pids_test.go#L42.  You don't need to do that in this PR unless you want to.

Reply:
At least for now I've left well enough alone.